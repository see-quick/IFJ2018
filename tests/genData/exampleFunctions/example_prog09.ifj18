# Print biggest number from three numbers

a = inputi
print "\n"
b = inputi
print "\n"
c = inputi
print "\n"

if c > b then
    if c > a then
        print "\nBiggest number is ", c, "\n"
    else
        print "\nBiggest number is ", a, "\n"
    end
else
    print "\nBiggest number is ", b, "\n"
end
