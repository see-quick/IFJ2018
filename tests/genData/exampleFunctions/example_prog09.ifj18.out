5
4
6

Biggest number is 6
